### DISCUSSION TITLE
Pandora Behaviour Engine v0.9.0-alpha

### DISCUSSION THREAD

**Discussion by Monitor221hz** [Monitor221hz/Pandora-Behaviour-Engine-Plus#48] [Announcements]
Deprecated graph injection (RIP)

Out-of-the-box FNIS support (Patches no longer needed for FNIS mods)

Minor bug fixes

Messaging improvements: 

- Displaying FATAL (engine failure) level errors in messagebox instead of just the log file.
- Displaying new animation count for every patched project and total.
- Displaying active FNIS and Nemesis/Pandora mods separately.

Graph injection may be gone, but now I know where to take development in terms of user demand. Some refactoring is in order, so this should be the last **major** release for a while.

<hr /><em>This discussion was created from the release <a href='https://github.com/Monitor144hz/Pandora-Behaviour-Engine-Plus/releases/tag/v0.9.0-alpha'>Pandora Behaviour Engine v0.9.0-alpha</a>.</em>

**Comment by lobotomy-x** [comment]
Could you elaborate on the implications of graph injection being deprecated for users?

**Reply by Monitor221hz** [reply]
For users, should no longer need poser mod compatibility patches as FNIS animlist mods are now fully compatible with FNIS
